### PR TITLE
Gerald feat

### PR DESCRIPTION
**Pull request by mamasalanang-gerald**
i added my module to my branch

### CODE REVIEW

File: errawrs/mamasalanang.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        return self.age
+
+    def major_property(self):
+        return self.major
+
+    def school_property(self):
+        return self.school
+
+    def greet(self):
+        print(f"Hello, my name is {self.name} and I am a student at {self.school}.")
```
Comment (victorioso-daniel-2) [inline errawrs/mamasalanang.py:21]: exceeding column 80

File: errawrs/mamasalanang.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    def birthday(self):
+        self.age += 1
+        print(f"Happy birthday! You are now {self.age} years old.")
+
+    def change_major(self, new_major):
+        self.major = new_major
+        print(f"Now I am studying {self.major}.")
+
+    def info(self):
+        print(f"Name: {self.name}, Age: {self.age}, Major: {self.major}, School: {self.school}")
```
Comment (victorioso-daniel-2) [inline errawrs/mamasalanang.py:35]: exceeding column 80

File: errawrs/mamasalanang.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    def birthday(self):
+        self.age += 1
+        print(f"Happy birthday! You are now {self.age} years old.")
+
+    def change_major(self, new_major):
+        self.major = new_major
+        print(f"Now I am studying {self.major}.")
+
+    def info(self):
```
Comment (victorioso-daniel-2) [inline errawrs/mamasalanang.py:34]: verb form for methods

File: errawrs/mamasalanang.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    def school_property(self):
+        return self.school
+
+    def greet(self):
+        print(f"Hello, my name is {self.name} and I am a student at {self.school}.")
+
+    def study(self):
+        print(f"I am studying {self.major} at {self.school}.")
+
+    def birthday(self):
```
Comment (victorioso-daniel-2) [inline errawrs/mamasalanang.py:26]: verb form for methods

File: errawrs/mamasalanang.py [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    def name_property(self):
+        return self.name
+
+    def age_property(self):
+        return self.age
+
+    def major_property(self):
+        return self.major
+
+    def school_property(self):
```
Comment (victorioso-daniel-2) [inline errawrs/mamasalanang.py:17]: verb form for methods

File: errawrs/mamasalanang.py [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+        self.major = "Information Technology" 
+        self.school = "PUP Taguig"  
+
+    def name_property(self):
+        return self.name
+
+    def age_property(self):
+        return self.age
+
+    def major_property(self):
```
Comment (victorioso-daniel-2) [inline errawrs/mamasalanang.py:14]: verb form for methods

File: errawrs/mamasalanang.py [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    def __init__(self):
+        self.name = "Gerald Mamasalanang"
+        self.age = 19
+        self.major = "Information Technology" 
+        self.school = "PUP Taguig"  
+
+    def name_property(self):
+        return self.name
+
+    def age_property(self):
```
Comment (victorioso-daniel-2) [inline errawrs/mamasalanang.py:11]: verb form

File: errawrs/mamasalanang.py [hunk 8]
```diff
@@ -0,0 +1,64 @@
+class Mamasalanang:
+    def __init__(self):
+        self.name = "Gerald Mamasalanang"
+        self.age = 19
+        self.major = "Information Technology" 
+        self.school = "PUP Taguig"  
+
+    def name_property(self):
```
Comment (victorioso-daniel-2) [inline errawrs/mamasalanang.py:8]: verb form

**Review by victorioso-daniel — CHANGES_REQUESTED** [review]
Follow coding guidelines

**Review by mosquito-michaelangelo — CHANGES_REQUESTED** [review]
follow guidelines

File: errawrs/mamasalanang.py (hunk 1 above)
Comment (mamasalanang-gerald) [reply errawrs/mamasalanang.py]: got that

File: errawrs/mamasalanang.py (hunk 2 above)
Comment (mamasalanang-gerald) [reply errawrs/mamasalanang.py]: oki noted

File: main.py [hunk 9]
```diff
@@ -19,6 +20,8 @@ def main_function():
             print("Ivan")
         elif choice == 2:
             print("Gerald")
+            student_instance = Mamasalanang()
+            student_instance.menu()
```
Comment (victorioso-daniel-2) [inline main.py:24]: calling not executed

**Review by victorioso-daniel — CHANGES_REQUESTED** [review]
change the calling of menu

File: main.py (hunk 9 above)
Comment (mosquito-michaelangelo) [inline main.py:24]: Change the calling of menu

**Review by mosquito-michaelangelo — CHANGES_REQUESTED** [review]

**Review by victorioso-daniel — APPROVED** [review]

**Review by mosquito-michaelangelo — APPROVED** [review]

**Pull request merged**